### PR TITLE
Readme enhanced

### LINKED ISSUE #40: Readme file enhancement
The Readme file can be enhanced by adding a list of all the Open Source programs in which this project was listed in.

### PR DESCRIPTION
**Pull request by anirudhsai20**
# Description:
Added a list of programming languages nand technologies used and a list of all the open source programs in which this project was listed in. 

Fixes #40 

## Type of change:
Documentation

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

 
# Checklist:

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] New and existing unit tests pass locally with my changes.
- [ ] Any dependent changes have been merged and published in downstream modules.

# Screenshots / Video:

### CODE REVIEW

**Comment by anirudhsai20** [comment]
I have not added the list of features as it was already present in the introduction section.  Please inform me if I have misunderstood about features. I will then make the necessary changes.

File: README.md [hunk 1]
```diff
@@ -101,6 +109,13 @@ We go by the moto
 <code><strong>Caffeinate☕|| Collaborate🤝🏼|| Celebrate🎊</strong></code>
 before that, please read <a href="https://github.com/flow2ml/Flow2ML/blob/main/CONTRIBUTING.md">contributing guidelines</a>
 
+## Open Source programs that Flow2ML is a part of:
```
Comment (chebroluharika) [inline README.md:112]: Please shift this entire block to upside Dependencies

File: README.md [hunk 2]
```diff
@@ -30,6 +30,14 @@ Check Installation and sample code to flow into your ML model much faster and ef
 <p>Flow2ML is an open-source library to make the machine learning process much simpler. It loads the image data and applies the given filters and returns train data, train labels, validation data, and validation labels.
 For all these steps it just takes 3 lines of code. It mostly helps beginners in the field of machine learning and deep learning where the user would deal with image-related data.</p>
 
+
+## Programming languages and technologies used:
```
Comment (chebroluharika) [inline README.md:34]: Include this point in Table of contents section

**Comment by anirudhsai20** [comment]
I have made the necessary changes. Please check.

**Review by chebroluharika — APPROVED** [review]

**Comment by chebroluharika** [comment]
LGTM

**Pull request merged**